apps/OpenSpace/ext/launcher/src/splitcombobox.cpp

```diff
@@ -66,14 +67,39 @@ void SplitComboBox::populateList(const std::string& preset) {
     // Clear the previously existing entries since we might call this function again
     clear();
 
+    // Create "icons" that we use to indicate whether an item is built-in or user content
+    QIcon userIcon = []() {
+        QPixmap px = QPixmap(20, 25);
+        px.fill(Qt::transparent);
+
+        QPainter painter = QPainter(&px);
+        painter.setBrush(QColor(183, 211, 149, 255));
+        painter.drawEllipse(0, 5, 19, 19);
+
+        QFont f = QFont("Arial");
+        f.setPixelSize(14);
+        f.setBold(true);
+        painter.setFont(f);
+        painter.drawText(0, 5, 20, 20, Qt::AlignCenter, "U");
+        return QIcon(px);
+    }();
```

Comment:
Maybe this icon should live somewhere else?  Thinking if we want to use it inside of the editor as well, for things that live in the user folder

Reply:
Moved the icon creation to a separate file.

The component came to be as a deduplication of the code that existed there before.  It's basically a combobox that can display to distinct lists of objects (hence SplitComboBox)